config/routes.rb

```diff
@@ -2209,6 +2209,7 @@
         tagging_edit
       ),
       :post => %w(
+        new
```

Comment:
You'll also need an edit route

Reply:
I want to open a new PR for edit (I had to do some changes in order for the edit to work so it quiet big). I think we should first finish with the 5 PRs I opened and then continue with other stuff (edit, dashboard and physical storage), but if you prefer I can open the edit PR now (the dashboard and physical storage changes depends on https://github.com/ManageIQ/manageiq-ui-classic/pull/7344 and https://github.com/ManageIQ/manageiq-ui-classic/pull/7343 but I think the edit is self contained).

Reply:
I opened PR for edit #7354